### ISSUE TITLE
Event subscribers are not being prioritised

### ISSUE THREAD

**Issue by delboy1978uk** [Kunstmaan/KunstmaanBundlesCMS#1873]
I had a problem getting a custom class implementing EventSubscriberInterface to trigger after one in the bundles.

`Kunstmaan\AdminBundle\EventListener\AdminLocaleListener` has the following method defining a priority of 17:

```php
    /**
     * getSubscribedEvents
     */
    static public function getSubscribedEvents()
    {
        return array(
            // Must be registered before the default Locale listener
            KernelEvents::REQUEST => array(array('onKernelRequest', 17)),
        );
    }
```

I set my priority lower to 10 in my class, but it was still triggering first. I then used the console to see what the priorities were set to:

![image](https://user-images.githubusercontent.com/2684575/37654597-04ad9168-2c43-11e8-993f-2c04e8bd60d8.png)

As you can see, the default of zero is still set. Also, setting a breakpoint in the static functions did not work, the code was not called. I guess that method is called if you manually set up the subscribers instead of using the symfony config?

Anyway, the solution is to have priority set in the tags defined in the configs, like so:

```yaml
my_website.preview_locale.listener:
        class: My\WebsiteBundle\EventListener\PreviewLocaleListener
        arguments: ['@translator',  '%kunstmaan_admin.default_admin_locale%']
        tags:
            - { name: 'kernel.event_listener', event: 'kernel.request', method: 'onKernelRequest', priority: -10 }
```

So I think we need to add this priority tag to all listeners/subscribers throughout the bundles where a custom priority has been set.

**Comment by acrobat** [comment]
Possibly fixed in #2692 as the eventsubscriber was registered as a listener, therefore the subscriber priority was ignored.

**Comment by krewetka** [comment]
I am using version 5.6.2 where priorities are like this:

![image](https://user-images.githubusercontent.com/207319/88569150-4c990a00-d03a-11ea-87f7-7531d89b9783.png)

and still expirience problem described here https://github.com/Kunstmaan/KunstmaanBundlesCMS/issues/2691

Problem is fixed when I have priority of AdminLocaleListener from 17 to -1 :thinking: 

![image](https://user-images.githubusercontent.com/207319/88570836-085b3900-d03d-11ea-91d2-3b0601c944f1.png)


@acrobat  do you have any idea why there is 17 and info  `// Must be registered before the default Locale listener` when it does set locale so the later the better in my opinion ( or otherwise other listeners overwrite it again)
